Lib/test/test_capi/test_opt.py

```diff
@@ -952,6 +952,276 @@ def testfunc(n):
         _, ex = self._run_with_optimizer(testfunc, 16)
         self.assertIsNone(ex)
 
+    def test_combine_stack_space_checks_sequential(self):
+        def dummy12(x):
+            return x - 1
+        def dummy13(y):
+            z = y + 2
+            return y, z
+        def testfunc(n):
+            a = 0
+            for _ in range(n):
+                b = dummy12(7)
+                c, d = dummy13(9)
+                a += b + c + d
+            return a
+
+
+        res, ex = self._run_with_optimizer(testfunc, 32)
+        self.assertEqual(res, 832)
+        self.assertIsNotNone(ex)
+
+        uops_and_operands = [(opcode, operand) for opcode, _, _, operand in list(ex)]
```

Comment:
The `list()` call is superfluous:
```suggestion
        uops_and_operands = [(opcode, operand) for opcode, _, _, operand in ex]
```
(Ditto below.)